rest_framework/utils/serializer_helpers.py

```diff
@@ -82,6 +83,12 @@ def as_form_field(self):
         return self.__class__(self._field, value, self.errors, self._prefix)
 
 
+class JSONBoundField(BoundField):
+    def as_form_field(self):
+        value = json.dumps(self.value)
```

Comment:
Does this display in a textarea or an input? Ideally an indented style and a textarea would probably be nice.